src/main/java/nl/hsac/fitnesse/junit/HsacFitNesseRunner.java

```diff
@@ -101,6 +102,9 @@ protected FitNesseContext createContext(Class<?> suiteClass) throws Exception {
     protected void runPages(List<WikiPage> pages, RunNotifier notifier) {
         boolean seleniumConfigOverridden = configureSeleniumIfNeeded();
         try {
+            if ("true".equalsIgnoreCase(System.getProperty(allureEnablerVariableName))){
```

Comment:
We could introduce a singleton (static field) to contain the list. The listener should be able to reference it from there. But that seems like a bit of a hack. 
Ideally we should be able to add the tags to the Description created for a page, that would be the preferred way to pass information. I'm not jUnit expert, but maybe we could have the runner add them as 'annotations'?